lib/modules/datasource/metadata-manual.ts

```diff
@@ -94,6 +94,10 @@ export const manualSourceUrls: Record<string, Record<string, string>> = {
   kubernetes: {
     node: 'https://github.com/nodejs/node',
   },
+  maven: {
+    'com.figure.gradle.semver-plugin:com.figure.gradle.semver-plugin.gradle.plugin': 
+      'https://github.com/FigureTechnologies/gradle-semver-plugin',
+  }
```

Comment:
```suggestion
  },
```

Reply:
@viceice fixed! thanks for the commit suggestion 👍